rmf_robot_sim_common/src/slotcar_common.cpp

```diff
@@ -243,6 +249,129 @@ void SlotcarCommon::path_request_cb(
   }
 }
 
+void SlotcarCommon::ackmann_path_request_cb(
```

Comment:
I think it would be helpful for future maintenance to document the steps of this function a bit (or break it down into subfunctions), I personally find it a bit hard to follow what is happening.

Reply:
yea there's a bit of math/geometry background needed here, add comments 1ec3bde